npm install error

### Expected Behavior

I wish to install the scratch-blocks in my Mac pro for my boy. 
Sorry for some error raised in the installation. 

### Actual Behavior

When I used the command "npm install" in the directory of "scratch-blocks" which a clone from github.com recently, some error raised.

```
Using local compiler: google-closure-compiler ...

SUCCESS: blockly_compressed_vertical.js
Size changed from 3083 KB to 939 KB (30%).
SUCCESS: blockly_uncompressed_vertical.js
SUCCESS: blockly_uncompressed_horizontal.js
SUCCESS: blockly_compressed_horizontal.js
Size changed from 3055 KB to 928 KB (30%).
SUCCESS: blocks_compressed_horizontal.js
Size changed from 39 KB to 19 KB (49%).
blocks_vertical/vertical_extensions.js:54: WARNING - Misplaced function annotation. This JSDoc is not attached to a function node. Are you missing parentheses?
  return function() {
  ^^^^^^^^^^^^^^^^^^^

0 error(s), 1 warning(s)
SUCCESS: blocks_compressed_vertical.js
Size changed from 173 KB to 87 KB (51%).
SUCCESS: blocks_compressed.js
Size changed from 21 KB to 8 KB (38%).
/Users/leeleo/Downloads/sourcecodes/scratch-blocks/node_modules/webpack-cli/bin/config-yargs.js:89
				describe: optionsSchema.definitions.output.properties.path.description,
				                                           ^

TypeError: Cannot read property 'properties' of undefined
    at module.exports (/Users/leeleo/Downloads/sourcecodes/scratch-blocks/node_modules/webpack-cli/bin/config-yargs.js:89:48)
    at /Users/leeleo/Downloads/sourcecodes/scratch-blocks/node_modules/webpack-cli/bin/webpack.js:60:27
    at Object.<anonymous> (/Users/leeleo/Downloads/sourcecodes/scratch-blocks/node_modules/webpack-cli/bin/webpack.js:515:3)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! scratch-blocks@0.1.0 prepublish: `python build.py && webpack`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the scratch-blocks@0.1.0 prepublish script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/leeleo/.npm/_logs/2018-10-14T15_59_58_301Z-debug.log
```

### Steps to Reproduce

git clone https://github.com/LLK/scratch-blocks.git
cd scratch-blocks
npm install

### Operating System and Browser

 Mac OS 10.11.6 Safari 10.0_

--- a/package.json
+++ b/package.json
@@ -42,7 +42,7 @@
     "travis-after-all": "1.4.4",
     "uglifyjs-webpack-plugin": "^1.2.5",
     "webdriverio": "4.8.0",
-    "webpack": "^4.8.0",
-    "webpack-cli": "^2.0.15"
+    "webpack": "^4.22.0",
+    "webpack-cli": "^3.1.1"
   }
 }

